Subject: No -DGDK_DISABLE_DEPRECATED
From: Michal Čihař <nijel@debian.org>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=631068

The -DGDK_DISABLE_DEPRECATED blocks GDK_DISPLAY in GTK 2.23.

This should be fixed better, see above referenced bug.

--- a/configure.ac
+++ b/configure.ac
@@ -147,7 +147,6 @@
 GPDS_CFLAGS="$XINPUT_CFLAGS $GTK_CFLAGS"
 GPDS_CFLAGS="$GPDS_CFLAGS $GCONF2_CFLAGS"
 GPDS_CFLAGS="$GPDS_CFLAGS -DGTK_DISABLE_DEPRECATED"
-GPDS_CFLAGS="$GPDS_CFLAGS -DGDK_DISABLE_DEPRECATED"
 GPDS_CFLAGS="$GPDS_CFLAGS -DG_DISABLE_DEPRECATED"
 GPDS_CFLAGS="$GPDS_CFLAGS -DGPDS_MODULEDIR=\\\"\"\$(gpds_moduledir)\"\\\""
 GPDS_CFLAGS="$GPDS_CFLAGS -DGPDS_DATADIR=\\\"\"\$(gpds_datadir)\"\\\""
